### PR TITLE
display infos when currency app is not supported by ledger live

### PR DESCRIPTION
**Pull request by valpinkman**
show a Learn More link when installing a cryptocyurrency app that is not supported by ledger live

Also reworked the whole modal when installing / uninstalling apps

### Type

UI Improvement

### Context

LL-1482

### Parts of the app affected / Test plan

Install supported app

image

Install unsupported app

image

Uninstall app

image

@khalilbenihoud @bscholving @dekkaki are the design ok witchu ?

### CODE REVIEW

File: src/components/ManagerPage/AppsList.js [hunk 1]
```diff
@@ -227,15 +268,20 @@ class AppsList extends PureComponent<Props, State> {
           </Box>
         )}
         <Text ff="Museo Sans|Regular" fontSize={6} color="dark">
-          {mode !== 'home' ? t(`manager.apps.${mode}`, { app }) : null}
+          {mode !== 'home' ? t(`manager.apps.${mode}`, { app: get(app, 'name', '') }) : null}
```
Comment (gre) [inline src/components/ManagerPage/AppsList.js]: I prefer we don't use `get(`. it is not typesafe. just use `app.name||""` (why is this needed?)

if app can be falsy, it's probably better to add the condition in the if so we never would render the wording with a falsy case 🤔

File: src/components/ManagerPage/AppsList.js [hunk 2]
```diff
@@ -227,15 +268,20 @@ class AppsList extends PureComponent<Props, State> {
...
         </Box>
       </Box>
     ) : status === 'error' ? (
       <Box>
-        <TrackPage category="Manager" name="Error Modal" error={error && error.name} app={app} />
+        <TrackPage
+          category="Manager"
+          name="Error Modal"
+          error={error && error.name}
+          app={get(app, 'name', '')}
```
Comment (gre) [inline src/components/ManagerPage/AppsList.js]: same

File: src/components/ManagerPage/AppsList.js [hunk 3]
```diff
@@ -264,9 +310,12 @@ class AppsList extends PureComponent<Props, State> {
...
     ) : status === 'success' ? (
       <Box grow align="center" justify="center" mt={5}>
-        <Box color="positiveGreen">
-          <CheckCircle size={44} />
-        </Box>
+        <IconWrapper>
+          <AppCheck>
+            <CheckFull size={24} />
+          </AppCheck>
+          <AppIcon src={this.getIconUrl()} />
```
Comment (gre) [inline src/components/ManagerPage/AppsList.js]: live-common already exposes a method for that, please use it:
https://github.com/LedgerHQ/ledger-live-common/blob/master/src/manager.js#L32

File: src/components/ManagerPage/AppsList.js [hunk 4]
```diff
@@ -275,17 +324,45 @@ class AppsList extends PureComponent<Props, State> {
...
+            color="smoke"
+            mt={2}
+            fontSize={4}
+            ff="Open Sans|Regular"
+            textAlign="center"
+            style={{ maxWidth: 350 }}
+          >
+            {t(`manager.apps.${isSupported ? 'supportedCurrency' : 'unsupportedCurrency'}`, {
+              app: app.name,
+              device: get(getDeviceModel(device.modelId), 'productName', ''),
```
Comment (gre) [inline src/components/ManagerPage/AppsList.js]: same

File: src/components/ManagerPage/AppsList.js [hunk 5]
```diff
@@ -211,13 +238,27 @@ class AppsList extends PureComponent<Props, State> {
...
+    const icon = app ? ICONS_FALLBACK[app.icon] || app.icon : ''
+    return `https://api.ledgerwallet.com/update/assets/icons/${icon}`
+  }
+
   renderBody = () => {
-    const { t } = this.props
+    const { t, device } = this.props
     const { app, status, error, mode, progress } = this.state
 
+    const isSupported = supported.includes(get(app, 'name', '').toLowerCase())
```
Comment (gre) [inline src/components/ManagerPage/AppsList.js]: same as comment below, we use this logic too often

File: src/components/ManagerPage/AppsList.js [hunk 6]
```diff
@@ -275,17 +324,45 @@ class AppsList extends PureComponent<Props, State> {
...
   }
 
   renderFooter = () => {
     const { t } = this.props
-    return (
+    const { app, mode } = this.state
+
+    const isInstalling = mode === 'installing'
+    const isCurrency = app && app.currency
+    const isSupported = app && supported.includes(app.name.toLowerCase())
```
Comment (gre) [inline src/components/ManagerPage/AppsList.js]: this does not seem correct to do `app.name.toLowerCase()` here. app name does not match app names but instead should match `app.currency.id`

so basically we should do this

```
isSupported = app && app.currency && isCurrencySupported(app.currency)
```

and probably introduce this `isCurrencySupported` instead of exporting `supported`

**Comment by gre** [comment]
@dasilvarosa it's just a bug in the implementation

**Comment by valpinkman** [comment]
> 1. Please change wording "Monero successfully installed" to: "Learn how to use Monero"

@dasilvarosa wording update
image

@gre @Arnaud97234 Fix implem to check on currency.id
image

image
